packages/react/src/primitives/Tabs/TabsContainer.tsx

```diff
@@ -20,7 +22,11 @@ const TabsContainerPrimitive: Primitive<TabsProps, 'div'> = (
   }: BaseTabsProps,
   ref
 ) => {
+  const groupId = useStableId(); // groupId is used to ensure uniqueness between Tab Groups in IDs
   const isControlled = controlledValue !== undefined;
+  if (defaultValue && typeof defaultValue === 'string') {
+    defaultValue = defaultValue.replace(' ', WHITESPACE_VALUE); // remove whitespace from defaultValue
```

Comment:
We can't trim the `defaultValue` since that os overwriting the value that the customer passed in

Reply:
That's the same thing we are doing to the values they pass in, this ensures the same change to the `defaultValue` that was done to the `value`. The original fix made it so that all `value` produced legal html IDs, but created the case where `defaultValue` also had whitespace and then could not reference the correct `value` generated ID.

Reply:
Modifying provided values breaks the contract of `onValueChange` - consumers cannot reliably trust that the values they have provided to the `Tabs` are the values that are provided to their `onValueChange` callback

Reply:
Ok, switched back to using `idValue = value.replace()` and doing nothing to `defaultValue`. Preserves user-inputted values while maintaining legal html IDs.